lms/envs/aws.py

```diff
@@ -500,3 +500,7 @@
 PDF_RECEIPT_COBRAND_LOGO_HEIGHT_MM = ENV_TOKENS.get(
     'PDF_RECEIPT_COBRAND_LOGO_HEIGHT_MM', PDF_RECEIPT_COBRAND_LOGO_HEIGHT_MM
 )
+
+if FEATURES.get('ENABLE_COURSEWARE_SEARCH'):
+    # Use ElasticSearch as the search engine herein
+    SEARCH_ENGINE = "search.elastic.ElasticSearchEngine"
```

Comment:
@andy-armstrong edxnotes is talking to an external data API with its own elasticsearch config behind the scenes, so not a shareable configuration for this case.